src/database/migrations/20220605123824-create-user.js

```diff
@@ -4,25 +4,43 @@ module.exports = {
   async up(queryInterface, Sequelize) {
     await queryInterface.createTable('Users', {
       id: {
+        type: Sequelize.UUID,
+        defaultValue: Sequelize.UUIDV4,
         allowNull: false,
-        autoIncrement: true,
-        primaryKey: true,
-        type: Sequelize.INTEGER
+        primaryKey: true
```

Comment:
do not touch old migration files.
Write a new migration file and do whatever updates!